### ISSUE TITLE
full_circuit example doesn't build on i686

### ISSUE THREAD

**Issue by junghans** [StanfordLegion/legion#182]
Error:
```
[  354s] /home/abuild/rpmbuild/BUILD/legion-legion-16.08.0/examples/full_circuit/circuit_cpu.cc:145:22: warning: SSE vector return without SSE enabled changes the ABI [-Wpsabi]
[  354s]  static inline __m128 get_vec_node_voltage(ptr_t current_wire,
[  354s]                       ^~~~~~~~~~~~~~~~~~~~
[  355s] In file included from /usr/lib/gcc/i586-suse-linux/6/include/x86intrin.h:33:0,
[  355s]                  from /home/abuild/rpmbuild/BUILD/legion-legion-16.08.0/examples/full_circuit/circuit_cpu.cc:18:
[  355s] /usr/lib/gcc/i586-suse-linux/6/include/xmmintrin.h: In function '__m128 get_vec_node_voltage(ptr_t, const LegionRuntime::Accessor::RegionAccessor<AT, float>&, const LegionRuntime::Accessor::RegionAccessor<AT, float>&, const LegionRuntime::Accessor::RegionAccessor<AT, float>&, const LegionRuntime::Accessor::RegionAccessor<AT_PTR, ptr_t>&, const LegionRuntime::Accessor::RegionAccessor<AT_VAL, PointerLocation>&) [with AT_VAL = LegionRuntime::Accessor::AccessorType::SOA<4u>; AT_PTR = LegionRuntime::Accessor::AccessorType::SOA<8u>]':
[  355s] /usr/lib/gcc/i586-suse-linux/6/include/xmmintrin.h:944:1: error: inlining failed in call to always_inline '__m128 _mm_set_ps(float, float, float, float)': target specific option mismatch

```
Details [here](https://github.com/StanfordLegion/legion/files/481646/_log.txt).

**Comment by streichler** [comment]
Should be fixed with c2e8cea.  @junghans can you verify on your side?

**Comment by junghans** [comment]
Works!
